lib/tasks/ingest.rake

```diff
@@ -389,6 +389,44 @@ namespace :ingest do
     end
   end
 
+  desc "Import all NYC schools"
+  # task import_all_nyc_schools: :environment do
+  task import_all_nyc_schools: :environment do
+    csv_text = File.read('data/2016_-_2017_School_Locations.csv')
```

Comment:
Why are we harding coding 'data/2016_-_2017_School_Locations.csv' ?

Reply:
I'll take your suggestion @nnat425 and change it, but the original reason was that passing in the CSV name as a variable to the rake task doesn't save much time.

The reason is that to run this rake task, we'll need the CSV to be available locally, so we will have to do a production deploy whenever there are new schools to add.  When we add the CSV, the developer would change this line too.  Having the file name in the code helps document where this CSV looks for its data in the file tree, but I can add that as a comment instead.